code/testapp/build.gradle.kts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("com.android.application")
+}
+
+val mavenCoreVersion: String by project
+
+android {
+      namespace = "com.adobe.campaignclassictestapp"
```

Comment:
nit: if you want to refactor the test app to 
com.adobe.marketing.mobile.<extension>.testapp

But since this is the unique namespace, so I think it is optional.

Reply:
Will take this in next PR